data-processing-lib/python/pyproject.toml

```diff
@@ -2,6 +2,7 @@
 name = "data_prep_toolkit"
 version = "0.2.0.dev6"
 requires-python = ">=3.10"
+keywords = ["data", "data preprocessing", "data preparation", "llm", "generative", "ai", "fine-tuning", "llmapps" ]
 description = "Data Preparation Toolkit Library"
 license = {text = "Apache-2.0"}
```

Comment:
Should it specify Python in the description and maybe name??

Reply:
In the future, please make a suggestion to your liking.  How aboute

`Data Preparation Toolkit for Python`

However, this seems a bit redundant since pypi sort of imlies python.

Reply:
We have 3 different libraries - one for Ray, one for Spark and one for ? nothing?

Reply:
I think the name of the library, `Data Preparation Toolkit Library` is clear enough.